qa/valgrind.supp

```diff
@@ -671,12 +671,7 @@
    Memcheck:Cond
    ...
    fun:EVP_DecryptFinal_ex
-   fun:_ZN4ceph6crypto6onwire25AES128GCM_OnWireRxHandler34authenticated_decrypt_update_finalEONS_6buffer7v*4listEj
-   fun:_ZN10ProtocolV231handle_read_frame_epilogue_mainEOSt10unique_ptrIN4ceph6buffer7v*8ptr_nodeENS4_8disposerEEi
-   fun:_ZN10ProtocolV216run_continuationER2CtIS_E
-   ...
-   fun:_ZN15AsyncConnection7processEv
-   fun:_ZN11EventCenter14process_eventsEjPNSt6chrono8durationImSt5ratioILl1ELl1000000000EEEE
+   fun:_ZN4ceph6crypto6onwire25AES128GCM_OnWireRxHandler34authenticated_decrypt_update_finalE?NS_6buffer7v*4listE*
```

Comment:
> i added 2 new wildcards to match both forms, and verified with local testing that it correctly suppresses the issue

I don't think new wildcards are needed.  `authenticated_decrypt_update_final()` signature was changed in commit fe97a000990632dbe9734e0d3a96df627f104f7a, it takes an lvalue reference and no longer has an integer parameter.

Apologies for missing to update the suppression there!

Reply:
thanks, removed the wildcards